app.js

```diff
@@ -35,28 +34,11 @@ app.use(async function(ctx, next) {
 const Router = require('koa-router');
 const router = new Router();
 
-const creatorKeyJson = (() => {
-    try {
-        return JSON.parse(process.env.ACCOUNT_CREATOR_KEY);
-    } catch (e) {
-        console.warn(`Account creation not available.\nError parsing ACCOUNT_CREATOR_KEY='${process.env.ACCOUNT_CREATOR_KEY}':`, e);
-        return null;
-    }
-})();
-const keyStore = {
-    async getKey() {
-        return nearAPI.KeyPair.fromString(creatorKeyJson.secret_key || creatorKeyJson.private_key);
-    }
-};
+const {
+    creatorKeyJson,
```

Comment:
Makes sense to extract all the code using `creatorKeyJson` into module and just export middleware handling account creation. Any reason not to do it?

Reply:
Seems like a majority of the near code should go in `middleware/near.js` vs. `utils/near.js` correct?

And we can tighten this: https://github.com/near/near-contract-helper/blob/d70186d6f37625eedf238e048c2a2042e143745f/app.js#L37
Into a proper middleware `withNear` as you mentioned in: https://github.com/near/near-contract-helper/pull/165/files/d70186d6f37625eedf238e048c2a2042e143745f#r445233017

Reply:
yeah, seems good to me